File: LNPG_01_TIPOS_E_VARIAVEIS/ATIVIDADE_3/JAVA/Main.java
package ATIVIDADE_3.JAVA;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        Thermometer term = new Thermometer();

        System.out.print("Digite a temperatura em °C: ");
        double c = scan.nextDouble();
        term.transform(c);

        scan.close();
    }
}

File: LNPG_01_TIPOS_E_VARIAVEIS/ATIVIDADE_3/C/main.c
#include <stdio.h>

int main() {
    double c;

    printf("Digite a temperatura em °C: ");
    scanf("%lf", &c);
  
    double f = c * 1.8 + 32;
    printf("A temperatura em °F é: %.2lf\n", f);
  
    return 0;
}